### DISCUSSION TITLE
How to chain commands and subcomands?

### DISCUSSION THREAD

**Discussion by marcellodesales** [ko1nksm/getoptions#18] [Q&A] [answered]
Hi @ko1nksm ... awesome library! thank you for making it! 

I have the following commands based on https://github.com/kubernetes/kubectl?

* `kubectl get pod xys -n namespace`
* `kubectl -n namespace describe service abc`
* `kubectl delete ingress 123`

Where:

* the global is `namespace` string defaulting to "default"
* sub_commands are `get`, `delete`, `describe`
* sub_sub_commands are `pod`, `ingress`, `service` subcommands of all of the first subcommands 
* `xyz`, `abc` and `123` are parameters to each 

How can I go about implementing this use-case of multiple commands and subcommands? Based on the example `subcmd` I don't see an option to provide that... This is related to #16 ....

**Comment by ko1nksm** [comment, marked as answer]
I believe I answered your question in #16.
